src/main/java/gov/usds/case_issues/model/BatchUpdateRequest.java

```diff
@@ -0,0 +1,46 @@
+package gov.usds.case_issues.model;
+
+import java.util.List;
+import java.util.Optional;
+
+public class BatchUpdateRequest {
+
+	private BatchUpdateAction updateAction;
+	private List<String> receiptNumbers;
+	private Optional<String> snoozeReason;
+	private List<AttachmentRequest> attachments;
+	private int duration;
```

Comment:
Inconsistent: make this an `Optional<Integer>`, and apply `@ApiModelParam` (?) to make Swagger OK with that.